api/src/controllers/document-controller.js

```diff
@@ -261,8 +264,8 @@ async function createURLDocument(fileName, addedBy, url, readRoles = []) {
   document.fileName = fileName;
   document.addedBy = addedBy;
   document.url = url;
-  document.read = ['sysadmin', ...readRoles];
-  document.write = ['sysadmin'];
+  document.read = [ROLES.SYSADMIN, ROLES.LNGADMIN, ROLES.NRCEDADMIN, ROLES.BCMIADMIN, ...readRoles];
+  document.write = ROLES.ADMIN_ROLES;
```

Comment:
This should probably match the function below.

Reply:
nice catch, pushed up a change